plugins/modules/email_alert.py

```diff
@@ -180,7 +180,7 @@ def delete_email_alert(module: AnsibleModule, rest_client: RestClient):
         dict(email_address=module.params["email"]), rest_client
     )
     if not delete_email:
-        module.warn("Email Alert: The email you're trying to remove, doesn't exist.")
+        module.warn("Email Alert: The email you're trying to remove doesn't exist.")
```

Comment:
No need to show warn if email was already removed. warn will signal to user something seems to be broken. But everything is OK.